### PR TITLE
add some basic print styling to lesson plans

### PR DESCRIPTION
**Pull request by Hamms**
Which allows them to be easily printed directly from the browser, or to be automatically turned into pdfs with [our existing pdf generation infrastructure](https://github.com/code-dot-org/code-dot-org/blob/staging/lib/pdf/conversion.rb)

Right now, the generated PDFs look something like:

[Lesson 1.pdf](https://github.com/code-dot-org/code-dot-org/files/5954897/Lesson.1.pdf)
[Lesson 2.pdf](https://github.com/code-dot-org/code-dot-org/files/5954898/Lesson.2.pdf)
[Lesson 3.pdf](https://github.com/code-dot-org/code-dot-org/files/5954899/Lesson.3.pdf)
[Lesson 4.pdf](https://github.com/code-dot-org/code-dot-org/files/5954900/Lesson.4.pdf)
[Lesson 5.pdf](https://github.com/code-dot-org/code-dot-org/files/5954901/Lesson.5.pdf)

## Testing story

I manually verified that this change doesn't affect the existing non-print styles at all. Once we have the ability to write UI tests for lesson plans, it would probably be good to add an eyes test. Added [PLAT-747] to track doing that

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

[PLAT-747]: https://codedotorg.atlassian.net/browse/PLAT-747

### CODE REVIEW

File: apps/style/lessons.scss [hunk 1]
```diff
@@ -1,6 +1,57 @@
...
+  /* Bootstrap adds the href of links to the print view by default; disable it
+   * for lessons. */
+  a[href]:after {
+    content: none !important;
+  }
+
+  /* Bootstrap coerces the color of _everything_ to be black, but for the
+   * progress bubble checkmarks this means we'd be drawing a black checkmark on
+   * a black background. We attempt to scope this to just stacked checkmarks. */
+  .fa-check.fa-stack-1x {
```
Comment (bethanyaconnor) [inline apps/style/lessons.scss:50]: Do we even want the checkmarks to be printed?

Comment (Hamms) [reply apps/style/lessons.scss:50]: good question! @tess323, thoughts?

Comment (tess323) [reply apps/style/lessons.scss:50]: I think since they indicate assessment (not completed) and since you have them looking nice we do

**Review by bethanyaconnor — APPROVED** [review]

**Comment by davidsbailey** [comment]
> I manually verified that this change doesn't affect the existing non-print styles at all. Once we have the ability to write UI tests for lesson plans, it would probably be good to add an eyes test. Added [PLAT-747](https://codedotorg.atlassian.net/browse/PLAT-747) to track doing that

We do have this ability: https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/test/ui/features/curriculum_platform/levelbuilder/lesson_edit_page.feature#L42

**Review by davidsbailey — APPROVED** [review]
This looks like a great start!

**Comment by Hamms** [comment]
> We do have this ability: https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/test/ui/features/curriculum_platform/levelbuilder/lesson_edit_page.feature#L42

Oh, I should've been more specific; I'm referring to the ability to write eyes tests against existing content. We certainly could get this to work via something that creates the content anew for each test run, but I'd much prefer to do something more traditional.

**Comment by davidsbailey** [comment]
> > We do have this ability: https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/test/ui/features/curriculum_platform/levelbuilder/lesson_edit_page.feature#L42
> 
> Oh, I should've been more specific; I'm referring to the ability to write eyes tests against existing content. We certainly could get this to work via something that creates the content anew for each test run, but I'd much prefer to do something more traditional.

Makes sense. I see your note in the jira item about allthecurriculumthings, which sounds like the right approach.

**Pull request merged**